### PR TITLE
Modify kind-gcr.sh to prevent the use of credsStore

### PR DESCRIPTION
**Pull request by MaxRenaud**

### CODE REVIEW

File: site/static/examples/kind-gcr.sh [hunk 1]
```diff
@@ -10,6 +10,16 @@ DOCKER_CONFIG=$(mktemp -d)
 export DOCKER_CONFIG
 trap 'echo "Removing ${DOCKER_CONFIG}/*" && rm -rf ${DOCKER_CONFIG:?}' EXIT
 
+echo "Creating a temporary config.json"
+# This is to force the omission of credsStore, which is automatically
+# created on supported system. With credsStore missing, "docker login"
+# will store the password in the config.json file.
+# https://docs.docker.com/engine/reference/commandline/login/#credentials-store
+cat <<EOF > /var/folders/1g/zx664t193xq9_bdmng9f50k800dbxr/T/tmp.htq9tIZy/config.json
```
Comment (BenTheElder) [inline site/static/examples/kind-gcr.sh:18]:
```suggestion
cat <<EOF >"${DOCKER_CONFIG}/config.json"
```

**Comment by MaxRenaud** [comment]
CLA signed

**Comment by BenTheElder** [comment]
/ok-to-test
The CNCF bot doesn't seem to know it yet :thinking:

File: site/static/examples/kind-gcr.sh [hunk 2]
```diff
@@ -27,4 +37,4 @@ for node in $(kind get nodes --name "${KIND_CLUSTER_NAME}"); do
   docker exec "${node_name}" systemctl restart kubelet.service
 done
 
-echo "Done!"
\ No newline at end of file
+echo "Done!"
```
Comment (BenTheElder) [inline site/static/examples/kind-gcr.sh:40]: this renders a spurious blank line in the docs which we haven't had time to fix

**Comment by MaxRenaud** [comment]
Refresh CLA

File: site/static/examples/kind-gcr.sh (hunk 2 above)
Comment (MaxRenaud) [reply site/static/examples/kind-gcr.sh:40]: Thanks!

**Comment by BenTheElder** [comment]
(skipping e2e, this changes docs)

**Pull request merged**